config/basic-auth/default/credentials_patch.yaml

```diff
@@ -13,12 +13,12 @@ spec:
           mountPath: "/opt/metal3/auth/ironic"
           readOnly: true
         - name: ironic-inspector-credentials
-          mountPath: "/opt/metal3/auth/ironic-inspector"
+          mountPath: "/opt/metal3/auth/ironic"
```

Comment:
Sorry if I'm missing something, why do we need to change the `mountPath`?

Reply:
this is absolutely a mistake, was not done on purpose
I'll change it back